oldlog.go

```diff
@@ -139,6 +138,29 @@ func SetLogLevel(name, level string) error {
 	return nil
 }
 
+// SetLogLevelRegex sets all loggers to level `l` that match expression `e`.
+// An error is returned if `e` fails to compile.
+func SetLogLevelRegex(e, l string) error {
+	lvl, err := LevelFromString(l)
+	if err != nil {
+		return err
+	}
+
+	rem, err := regexp.Compile(e)
+	if err != nil {
+		return err
+	}
+
+	loggerMutex.Lock()
+	defer loggerMutex.Unlock()
+	for name := range loggers {
+		if rem.MatchString(name) {
```

Comment:
Do we want to know if the regex fails to match any of the subsystems?

Reply:
Good question. I would be willing to log an Info message for this case, since an error seems incorrect here.

Reply:
on second thought the lack of match _should_ be obvious by the logging output not changing.